site/content/docs/04-preloading.md

```diff
@@ -83,7 +83,12 @@ To fix this, Sapper provides `this.fetch`, which works on the server as well as
 </script>
 ```
 
-Note that you will need to use session middleware such as [express-session](https://github.com/expressjs/session) in your `app/server.js` in order to maintain user sessions or do anything involving authentication.
+It is important to note that `preload` may run on either the server or in the client browser. Code called inside `preload` blocks:
+  - should run on the same domain as any upstream api servers requiring credentials; otherwise `credentials: 'include'` cannot guarantee access to 3rd party session cookies
```

Comment:
shouldn't `credentials: 'include'` send the cookies even to other domains?

Reply:
@benmccann Doesn't that depend on how a project has their CORS settings configured? I'm pretty new to NodeJS so I might be mistaken

Reply:
cookies can't be cross-domain. Cookies from `.domain.com` will only ever be sent to `.domain.com`